src/status_im/ui/screens/activity_center/views.cljs

```diff
@@ -170,20 +177,19 @@
 (defn header
   []
   (let [screen-padding 20]
-    ;; TODO: Remove temporary (and old) background color when the screen and
-    ;; header are properly blurred.
-    [rn/view {:background-color (:ui-background @quo.colors/theme)}
+    [rn/view
```

Comment:
FYI: @smohamedjavid, the sticky header becomes unusable without a solid or blurred background. I know you're probably aware there's a separate issue to use a blurred background in the sticky header (https://github.com/status-im/status-mobile/issues/14131)

<img src="https://user-images.githubusercontent.com/46027/196172037-34908247-e34f-446f-8ee4-a6ad4bb285e4.png" height="400" />

Reply:
Made the header sticky by detaching the header from the flatlist.

Reply:
Great news it worked!